webapp/src/blocks/card.ts

```diff
@@ -7,7 +7,7 @@ import {IBlock, MutableBlock} from './block'
 interface Card extends IBlock {
     readonly icon: string
     readonly isTemplate: boolean
-    readonly properties: Readonly<Record<string, string>>
+    readonly properties: Readonly<Record<string, string | string[]>>
```

Comment:
One thing to be careful about is supporting changing the type of a property from Select <> Mult-select. Going from single-select to multi-select should appear seamless to the user (though there may be quite involved conversion code).

Going from multi-select to single-select should retain the options, but make the value the first value in the multi-select list.

Reply:
@chenilim on the same topic, I think we should also show a confirmation model when we convert property types between incompatible types. For example, a `person` type to an `updated at` type. This will avoid any surprises and data loss.

Reply:
When switching between `Single Select` and `Multi Select`, should we be retain the options? Right now we're not.

Reply:
Yes, to clarify:
* When changing from single-select to multi-select: The options and selected value should remain the same (only change is the user can specify additional values).
* When changing from multi-select to single-select: Retain the options, but make the value the first value in the multi-select list.

I'll leave it to @hahmadia, but both of these can be done in a subsequent checkin if necessary.

Reply:
Done